hbase-base/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM gradiant/hadoop-base:2.7.7 as hadoop
+FROM quay.io/kaszpir/dockerized-hadoop:3.1.3 as hadoop
```

Comment:
We must not hardcode an external registry in the Dockerfile.
We can use docker build args for that:
```
ARG hadoop_image="gradiant/hdfs:3.1.2"
FROM $hadoop_image as hadoop
...
```
Then just set hadoop_image variable in your build environment (for example):
```
docker build -t quay.io/kaszpir/hbse:2.1.10 --build-arg hadoop_image=quay.io/kaszpir/dockerized-hadoop:3.1.3 .
```

Also if you need hadoop 3.1.x versions, we have created dockerhub gradiant/hdfs repo with 2.7.7, 2.9.2, 3.1.2 and 3.2.1 images from current gradiant/dockerized-hadoop.

Reply:
yeah I know, that's why it's WIP/Draft.
I'm experimenting with some other options, but ARG looks the best.
Was trying to avoid any external Makefiles and so on because I wanted to use ofifical quay.io building image feature from Dockerfile.